modules/core/src/weaver/Expectations.scala

```diff
@@ -149,6 +149,13 @@ object Expectations {
       case Invalid(_) => success
     }
 
+    implicit class StringOps(str: String) {
+      def only(implicit loc: SourceLocation): TestName =
```

Comment:
Because integrating with junit is not sufficient to run individual tests from UI, this acts as poor-man's alternative by tagging the test. If such a tag is present in at least one test, only tests tagged with it are run.